packages/teams-js/src/public/dialog.ts

```diff
@@ -16,21 +18,45 @@ import { runtime } from './runtime';
  *
  * @beta
  */
+export interface Result {
```

Comment:
I know you took this name (Result) from some of the docs we looked at last week, but do you think there's a better name we could use? Result.result feels a little odd. Maybe something like SdkResponse? ErrorOrResult?